### PR TITLE
Preprod: movie: Dreamcatcher (2003) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Your typical Stephen King paranormal suspense stuff. Rating 3/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 1]
```diff
@@ -2932,6 +2932,51 @@ biological computer [Dr. Yuhara explained that Mechagodzilla was installed with
...
+3 <paul>
+
+:: Major Themes
+friendship [The story follows four male friends as they increasingly find themselves at the center of an invasion of parasitic aliens while otherwise minding their own business on their annual hunting trip in Maine.]
+telepathic ability [The four friends had all acquired telepathic powers after having saved a mentally disabled boy from bullies back when they were children. Their using of this special power was explored throughout the story.]
+mind reading ability [The four friends had all acquired the power to read minds after having saved a mentally disabled boy from bullies back when they were children. Their using of this special power was explored throughout the story.]
+extraterrestrial being [The large humanoid alien Mr. Gray was trying to contaminate the Boston water supply with parasitic alien larvae.]
+attack from outer space [Maine was the site of an invasion of parasitic aliens.]
+human vs. monster [Horrible alien worm creatures were going are parasitizing people.]
+monster [Horrible alien worm creatures were going are parasitizing people. In the end, the surviving friends faced off against a large alien worm creature.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:2963]: "were going are"?

Comment (odinlake) [reply notes/stories/film/film-scifi-2000s.st.txt:2963]: Also sounds like a motivation for "human vs. ..."

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 2]
```diff
@@ -2932,6 +2932,51 @@ biological computer [Dr. Yuhara explained that Mechagodzilla was installed with
...
+covert military operation [Colonel Curtis was in charge of a secret military operation to exterminate the invading extraterrestrials.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:2964]: Like with the "government secrecy" I mentioned in another review, I wonder if the "covertness" was really major here..?

Comment (odinlake) [reply notes/stories/film/film-scifi-2000s.st.txt:2964]: Maybe we need a theme for just military operation?

**Review by odinlake — APPROVED** [review]

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt]: Whoops. Changed "were going are" to "were going around". Typo...

And we have **human vs. monster** in there, too. So we're covered on the "human vs. ..." front.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:2964]: Hum... they made a pretty big deal about there being an elite military unit specializing in extraterrestrials.

But in general, I bet a **military operation** theme would come in handy.

**Comment by odinlake** [comment]
Even in your apologia “elite military unit specializing in
extraterrestrials” you fail to motivate the “covert” attribute. If the
theme was “elite military unit” we should use that. If it was “specialists
in extraterrestrials”, then that. I don’t like it when we segway into
similar sounding themes because our imagination has been trained (perhaps
by tinsel town) to expect co-occurence.

**Pull request merged**